pkgs/tools/security/sshguard/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchgit, autoreconfHook, yacc, flex, pkgs}:
+
+
+stdenv.mkDerivation rec {
+  version = "1.7.1";
+  name = "sshguard-${version}";
+
+  src = fetchgit {
+    url = "https://bitbucket.org/sshguard/sshguard.git";
+    rev = "8600fd5b70cd80fad87c414fd7c410082991a209";
+    sha256 = "1i2ic8nca4f61pl2ggjp4x4an37hhvjgk2wv4smjsj6lvz3bw06a";
+  };
```

Comment:
Why using `fetchgit` when you can get the stable release source from their SourceForge repo using `fetchurl` ?
https://sourceforge.net/projects/sshguard/files/sshguard/